Description: fix a FTBFS with GLIBC 2.28.
 Patch from Gentoo Linux. Thanks.
Author: Mike Frysinger <vapier@gentoo.org>
Bug-Debian: https://bugs.debian.org/915987
Origin: https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=f503f77079c568cd9628571cf69b418a51967e49
Reviewed-By: Joao Eriberto Mota Filho <eriberto@debian.org>
Last-Update: 2016-04-19
Index: partimage-0.6.9/src/client/misc.h
===================================================================
--- partimage-0.6.9.orig/src/client/misc.h
+++ partimage-0.6.9/src/client/misc.h
@@ -36,7 +36,7 @@ struct COptions;
 #endif
 
 #ifndef makedev
-  #define makedev(maj,min) (((maj) << 8) | min))
+  #define makedev(maj,min) (((maj) << 8) | (min))
 #endif
 
 // =======================================================
